### ISSUE TITLE
Automatically log meditation time on https://meditation.sirimangalo.org

### ISSUE THREAD

**Issue by lodenrogue** [yuttadhammo/BodhiTimer#15]
It would be nice to have a feature that when the timer goes off it gives the option to log meditation time on https://meditation.sirimangalo.org

**Comment by olastor** [comment]
Maybe it would be possible to implement this the other way around and start the timer a timer via link? The [Meditation+](https://gitlab.com/sirimangalo/meditation-plus) webapp is lacking a reliable timer because it's only running in the browser. If there was a feature to open Bodhi Timer and start the timer with a specific duration from within the web application, we could integrate Bodhi Timer in this way and improve the experience for Android users. 

The use cases would be:

- Start timer for only "walking" or "sitting": The url would provide one duration, timer does not repeat itself
- Start timer for "walking" and "sitting": The url would provide two durations, which would run in sequence (two timers after each other)

If the second option cannot be implemented very easily, it would for many cases also suffice if you can enable the repeat feature via url (since often the values for "walking" and "sitting" are the same).

**Comment by lodenrogue** [comment]
Sounds interesting. A few questions. 
Would the results then be logged in meditation plus after?
Is there a way in android to request the open to open from a browser with given params (walking, sitting, duration, etc)?

**Comment by olastor** [comment]
> Would the results then be logged in meditation plus after?

It would be logged before opening the timer. The sessions get stored on the server when you start a new session.

> Is there a way in android to request the app to open from a browser with given params (walking, sitting, duration, etc)?

I think it should be possible, but I am also not very familiar with building Android apps. Perhaps this is the correct resource: https://developer.android.com/training/basics/intents/filters

**Comment by lodenrogue** [comment]
That looks right to me. Sounds like a great idea!

**Comment by tzugen** [comment]
Yes, I think it would be possible to register an URL scheme, which will open the app and start with a certain configuration.
I can support pull requests if people want to implement it, but won't be able to do it myself in the coming time.

**Comment by olastor** [comment]
@tzugen Will 6.0.0 be released on Play Store?
